src/pages/index.tsx

```diff
@@ -84,7 +88,7 @@ export default function Home({ categories }: HomeProps) {
           <input
             type="text"
             className={styles.filterBoxInput}
-            placeholder="Search for icons..."
+            placeholder={`Search ${totalIconCount} Icons...`}
```

Comment:
```suggestion
            placeholder={`Search ${totalIconCount} Icons…`}
```

Let's use the proper ... character